src/Uno.UWP/Storage/KnownFolders.Android.cs

```diff
@@ -0,0 +1,14 @@
+﻿#if __ANDROID__
+
+namespace Windows.Storage
+{
+	public static partial class KnownFolders
+	{
+		internal static StorageFolder FolderFromAndroidName(string name) => new StorageFolder(Android.OS.Environment.GetExternalStoragePublicDirectory(name).CanonicalPath);
+		public static StorageFolder MusicLibrary { get => FolderFromAndroidName(Android.OS.Environment.DirectoryMusic); }

```

Comment:
```suggestion
		public static StorageFolder MusicLibrary => FolderFromAndroidName(Android.OS.Environment.DirectoryMusic);
```

Same for the line below